app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -230,7 +230,7 @@ def available_storage_domains(cluster_id = nil)
     end
 
     def storage_domains(opts = {})
-      client.storage_domains({:role => 'data'}.merge(opts))
+      client.storage_domains({:role => %w'data volume'}.merge(opts))
```

Comment:
I find `%w'a b'` for `['a', 'b']` highly unreadable. can this be `%w(data volume)` or `['data', 'volume']`?

but maybe that's just me :)

Reply:
@evgeni sure, changed it.